Makefile

```diff
@@ -2,39 +2,36 @@ BIN := mackerel-container-agent
 VERSION := 0.0.3
 REVISION := $(shell git rev-parse --short HEAD)
 
+export GO111MODULE=on
+
 .PHONY: all
 all: clean build
 
 BUILD_LDFLAGS := "\
 	-X main.version=$(VERSION) \
 	-X main.revision=$(REVISION)"
 
-.PHONY: deps
-deps:
-	go get -d ./...
-
 .PHONY: build
-build: deps
+build:
 	go build -ldflags=$(BUILD_LDFLAGS) -o build/$(BIN) ./cmd/$(BIN)/...
 
-.PHONY: test-deps
-test-deps:
-	go get -d -t ./...
-	go get -u golang.org/x/lint/golint
-
 .PHONY: test
-test: test-deps
+test:
 	go test -v ./...
 
+.PHONY: lint-deps
+lint-deps:
+	GO111MODULE=off go get golang.org/x/lint/golint
+
 .PHONY: lint
-lint: test-deps
+lint: lint-deps
 	go vet ./...
 	golint -set_exit_status ./...
 
 .PHONY: clean
 clean:
 	rm -fr build
-	go clean
+	GO111MODULE=off go clean
```

Comment:
Why are you disabling go module here?

Reply:
In the case of GO111MODULE = on, the following error will occur.

```
$ go clean
can't load package: package github.com/mackerelio/mackerel-container-agent: unknown import path "github.com/mackerelio/mackerel-container-agent": cannot find module providing package github.com/mackerelio/mackerel-container-agent
```

Reply:
Ah, try `go clean ./...`.

Reply:
Thank you!